vaadin-combo-box-flow-parent/vaadin-combo-box-flow-integration-tests/src/main/java/com/vaadin/flow/component/combobox/test/template/ComboBoxTemplateInitialValue.java

```diff
@@ -0,0 +1,71 @@
+package com.vaadin.flow.component.combobox.test.template;
+
+import com.vaadin.flow.component.Tag;
+import com.vaadin.flow.component.combobox.ComboBox;
+import com.vaadin.flow.component.dependency.JsModule;
+import com.vaadin.flow.component.littemplate.LitTemplate;
+import com.vaadin.flow.component.template.Id;
+import com.vaadin.flow.data.binder.Binder;
+
+import java.io.Serializable;
+import java.util.ArrayList;
+import java.util.List;
+
+@JsModule("./src/combobox-template-initial-value-serverside.ts")
+@Tag("combobox-initial-value")
+public class ComboBoxTemplateInitialValue extends LitTemplate {
+
+    @Id("combo")
+    private ComboBox<Bean> comboBox;
+
+    List<Bean> beans = new ArrayList<>();
```

Comment:
New Sonarqube issue: CRITICAL ComboBoxTemplateInitialValue.java#L21: Make "beans" private or transient. rule

Reply:
Done.

Reply:
This is not resolved, the Sonarqube issue refers to the `beans` field that is marked in the code above.

In general please always take a look at Sonarqube issue to check if your new code introduced them.